webcomponents/tool/src/main/frontend/packages/sakai-announcements/src/SakaiAnnouncements.js

```diff
@@ -98,6 +99,14 @@ export class SakaiAnnouncements extends SakaiPageableElement {
           return 0;
         });
         break;
+      case INSTRUCTOR_ORDER:
+        this.data.sort((a1, a2) => {
+
+          if (a1.order < a2.order) return 1;
+          if (a1.order > a2.order) return -1;
+          return 0;
+        });
```

Comment:
```suggestion
        this.data.sort((a, b) => b.order - a.order);
```

Reply:
@ottenhoff message orders are stored in reverse in ANNOUNCEMENT_MESSAGE, for some reason. I didn't want to touch that code in announcements in case there were unforeseen consequences. It's annoying, for sure.